.github/release-drafter.yml

```diff
@@ -0,0 +1,52 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '# Breaking Changes'
+    labels:
+        - 'breaking change'
+  - title: '# Enhancements'
+    labels:
+      - 'type: enhancement'
+  - title: '# Performance'
+    labels:
+      - 'type: performance'
+  - title: '# Bugs Fixed'
+    labels:
+      - 'type: bug'
+  - title: '# Types'
+    labels:
+      - 'type: types'
+  - title: '# Documentation'
+    labels:
+      - 'type: documentation'
+  - title: '# Development'
+    labels:
+      - 'type: chore'
+change-template: '- #$NUMBER $TITLE'
+change-title-escapes: '\<*_&`#@'
+version-resolver:
+  major:
+    labels:
+      - 'breaking change'
+  minor:
+    labels:
+      - 'type: enhancement'
+  patch:
+    labels:
+      - 'type: chore'
+      - 'type: types'
```

Comment:
should this include bugs?

Reply:
Could add that, but anything not matched falls to the `default` below (patch)